package.nls.json

```
@@ -24,11 +24,11 @@
     "AWS.telemetry.notificationMessage": "Please help improve the AWS Toolkit by enabling it to send usage data to AWS. You can always change your mind later by going to the \"AWS Configuration\" section in your user settings.",
     "AWS.telemetry.notificationYes": "Enable",
     "AWS.telemetry.notificationNo": "Disable",
-    "AWS.command.github": "View Source on Github",
+    "AWS.command.github": "View Source on GitHub",
     "AWS.command.help": "View Documentation",
     "AWS.command.login": "Connect to AWS",
     "AWS.command.logout": "Sign out",
-    "AWS.command.reportIssue": "Report an Issue",
+    "AWS.command.createIssueOnGitHub": "Create a New Issue on GitHub",
```

Gratuitous Title Case.
```suggestion
    "AWS.command.createIssueOnGitHub": "Create a new issue on GitHub",
```

---

I'm in favor of this, but this is the casing of the JetBrains plugin 🤔

---

Are we using jetbrains as the authority for all toolkits? That could propagate accidental patterns...

---

It would certainly be better to define explicit patterns for casing, etc. across the board.
If we did that, I'd be in favor of title case since it downplays the less important words.

As it stands, it looks like the casing is each is inconsistent (even with itself).
My intent is just to align on a single phrasing. The jetbrains phrasing is more precise and consistent with the other phrasing in vscode (`View Source on Github` versus `View Source`)

---

> Are we using jetbrains as the authority for all toolkits? That could propagate accidental patterns...

burn.

---

Looks like both jetbrains and vscode use TitleCase for context menus. We should update the other vscode context menu strings to use this as well.

---

I For One Welcome Our New Titlecase Overlords.